src/cursor/cur_file.c

```diff
@@ -117,12 +117,12 @@ err:	API_END_RET(session, ret);
 }
 
 /*
- * __curfile_next_random --
+ * __wt_curfile_next_random --
  *	WT_CURSOR->next method for the btree cursor type when configured with
  * next_random.
```

Comment:
I'd add a comment to the effect "exported because it is called directly for LSM cursors"